tests/uc-services_apps-sap_build_apps.tfvars

```diff
@@ -0,0 +1,17 @@
+# ------------------------------------------------------------------------------------------------------
+# Provider configuration
+# ------------------------------------------------------------------------------------------------------
+# Region for your subaccount
+region        = "us10"
+
+# ------------------------------------------------------------------------------------------------------
+# Project specific configuration
+# ------------------------------------------------------------------------------------------------------
+# Account setup
+subaccount_name = "Int-test: SAP Build Apps"
```

Comment:
To stay consistent, I would stick to "Integration Test - SAP Build Apps" as subaccount name

Reply:
Makes sense